src/Api/Billing/Controllers/ProviderBillingController.cs

```diff
@@ -28,17 +28,17 @@ public async Task<IResult> GetSubscriptionAsync([FromRoute] Guid providerId)
             return TypedResults.Unauthorized();
         }
 
-        var subscriptionData = await providerBillingQueries.GetSubscriptionData(providerId);
+        var providerSubscriptionDTO = await providerBillingQueries.GetSubscriptionDTO(providerId);
```

Comment:
Everything in this file is renames to better organize how we refer to specific types of models.